### DISCUSSION TITLE
awesome-flynt – a collaborative space for sharing components and features

### DISCUSSION THREAD

**Discussion by timohubois** [flyntwp/flynt#529] [Show and tell]
Hey guys, a while ago an idea came up: "create a collaborative space for sharing components and features". Personally I think this is a great idea but to make this possible there is a lot what need to be done.

As a starting point, I created a [flynt-add-ons](https://github.com/awesome-flynt/flynt-add-ons) repository and added 3 features to it – no better idea where to start 😅

The added features are experimental and not part of the core of Flynt itself due Flynt is boilerplate-driven, should stay lean and be a starting point for new projects, but maybe they help or inspire someone ✌️

What do you think about this idea? I'm open to everything and happy to spend some of my free time on such a project.

Looking forward to your feedback and/or contributions 🚀

<hr>

Edit: created an organization ([awesome-flynt](https://github.com/awesome-flynt)) and moved the [repository](https://github.com/awesome-flynt/flynt-add-ons) to it; also updated the link above.

**Comment by aaronmeder** [comment]
That is a great idea @timohubois

**Reply by timohubois** [reply]
@aaronmeder nice to read, that you like the idea. When I’m not wrong you created a video about Flynt, aren’t you? What do you think about to place it with link into the [README.md](https://github.com/awesome-flynt/awesome-flynt/blob/main/README.md) of the repository?

**Comment by vitalijalbu** [comment]
Hi guys, anyone can share some websites made with **flynt**?

**Reply by timohubois** [reply]
@vitalijalbu there is a other [discussion](https://github.com/flyntwp/flynt/discussions/380) with some input, but I think in the wild there a lot more websites build with Flynt and we can also add a section “Build with Flynt” to the [awesome-flynt](https://github.com/awesome-flynt/awesome-flynt) repository. Feel free the bring in all your ideas and contribute.

**Reply by vitalijalbu** [reply]
will you guys release something for drupal in future maybe?? ...

**Reply by vitalijalbu** [reply]
I would like to buy the script, but I want to see the rpeview also of blocked compnents...

**Reply by timohubois** [reply]
@vitalijalbu for drupal there is nothing planned atm.

To get examples of components, you can take here:
https://flyntwp.com/components/

**Comment by rajpatel2435** [comment]
Hi everyone,


I’m excited to share clone-wp-theme, a new CLI tool designed to streamline WordPress theme setup. With this tool, you can easily choose and clone predefined themes like Flynt and Sage-Vite.

Features:
Interactive Setup: Choose from Flynt or Sage-Vite and automatically clone the corresponding repo.

Simple Installation: Install globally with npm install -g clone-wp-theme and start with clone-wp-theme

Check out the [README](https://github.com/rajpatel2435/create-wp-theme) for more details and instructions.

https://www.npmjs.com/package/clone-wp-theme

I’d love your feedback and contributions. Let’s make theme development easier together!

Best,
Raj Patel

**Reply by timohubois** [reply]
@rajpatel2435 this is a really cool idea! Streamlining the WordPress theme setup with a tool like clone-wp-theme makes it easier for developers to get started. We currently use other workflows, but this CLI tool could definitely be a time-saver.

I’m thinking of adding this to the README of the [awesome-flynt/flynt-add-ons](https://github.com/awesome-flynt/flynt-add-ons) repo so others can easily find and try it out. What do you think about this idea?

Great work, and looking forward to seeing this tool grow!

**Reply by rajpatel2435** [reply]
Hi @timohubois @aaronmeder @steffenbew 

Thank you for your support! I’m thrilled you find clone-wp-theme helpful. Integrating it directly into the main Flynt theme would be an exciting opportunity and could streamline theme setup even more for Flynt users.

I’d be happy to explore contributing this tool to the core Flynt theme itself. It would allow developers to kickstart projects even faster, all within the Flynt framework. If the Flynt team is open to it, I can create a pull request, and we can work together to make it fit seamlessly with the existing workflow.

Let me know your thoughts on this, and I’m excited about the possibility of contributing directly to the Flynt theme!

Best regards,
Raj Patel
